dev/MRTCore/mrt/Core/unittests/UnitTest.vcxproj

```diff
@@ -346,7 +347,17 @@
       <DeploymentContent Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">true</DeploymentContent>
     </CopyFileToFolders>
   </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
+  </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-  <ImportGroup Label="ExtensionTargets">
+    <ImportGroup Label="ExtensionTargets">
+    <Import Project="..\..\packages\Microsoft.Taef.10.58.210222006-develop\build\Microsoft.Taef.targets" Condition="Exists('..\..\packages\Microsoft.Taef.10.58.210222006-develop\build\Microsoft.Taef.targets')" />
   </ImportGroup>
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
```

Comment:
>EnsureNuGetPackageBuildImports [](http://example.com/codeflow?start=16&length=30)

Just curious Jeff, did you add this to help the dev in case the package was absent? Instead of them getting a possibly mysterious error or something?

Reply:
I actually copied this from the TAEF sample test app -- but you're quite right, it gives a more user-friendly error message if the Microsoft.Taef nuget package isn't present. :)